codec-http/src/main/java/io/netty/handler/codec/http/cookie/ClientCookieDecoder.java

```diff
@@ -248,11 +248,18 @@ private void parse8(int nameStart, int valueStart, int valueEnd) {
             if (header.regionMatches(true, nameStart, CookieHeaderNames.HTTPONLY, 0, 8)) {
                 httpOnly = true;
             } else if (header.regionMatches(true, nameStart, CookieHeaderNames.SAMESITE, 0, 8)) {
-                sameSite = SameSite.of(computeValue(valueStart, valueEnd));
+                String sameSiteValue = computeValue(valueStart, valueEnd);
+                try {
+                    sameSite = SameSite.of(sameSiteValue);
+                } catch (IllegalArgumentException e) {
+                    if (logger.isDebugEnabled()) {
```

Comment:
Useless. The purpose of such test would be to avoid message resolution (eg String concatenation) when logging level is disabled. But there's no resolution here as you use the {} interpolation.
Please remove.

Reply:
Hi, I have modified that.  

*Please note I followed examples from some files in your code that happened to check the logging level (e.g, EpollSocketTestPermutation) - maybe you can at some point fix all of those so it's easier to follow project standards.